fern/openapi/openapi.json

```diff
@@ -1505,6 +1649,20 @@
         },
         "security": [{ "HTTPBearer": [] }]
       }
+    },
+    "/v1/monitoring/health": {
+      "get": {
+        "tags": ["monitoring"],
+        "summary": "Health Check",
+        "description": "Checks the health the service.\n\nIt returns 200 if the service is healthy.",
+        "operationId": "health_check",
+        "responses": {
+          "200": {
+            "description": "Successful Response",
+            "content": { "application/json": { "schema": {} } }
+          }
+        }
+      }
```

Comment:
Is this something we want to expose rn?

Reply:
should not be there, and should be internal — to unblock spartan, we can merge it in for this SDK update, and fix it as a fast follow + also not update the API reference

also open to quickly fixing and redeploying

Reply:
Yeah, could just hand edit to remove from the json for now?